docs/copied-from-beats/command-reference.asciidoc

```diff
@@ -15,30 +15,32 @@
 :global-flags: Also see <<global-flags,Global flags>>.
 
 :deploy-command-short-desc: Deploys the specified function to your serverless environment
+
+ifndef::no_dashboards_apm[]
```

Comment:
L19-L21 should be removed?
Looks a dup of L23-25, same thing happens on several places bellow

Reply:
L19-L21 is an `ifndef` (if not defined) - this line shows up *outside* of the APM docs
L23-25 is an `ifdef` (if defined) - this line shows up *in* the APM docs